scripts/st2bootstrap-el7.template.sh

```diff
@@ -231,6 +235,11 @@ install_st2() {
   sudo crudini --set /etc/st2/st2.conf database username "stackstorm"
   sudo crudini --set /etc/st2/st2.conf database password "${ST2_MONGODB_PASSWORD}"
 
+  # Configure [messaging] section in st2.conf (username password for RabbitMQ access)
+  RABBITMQHOST="${RABBITMQHOST:-rabbitmq}"
```

Comment:
Can you change the default value to 127.0.0.1?

Reply:
+1 I also think we don't need to allow configuring the `RABBITMQHOST` outside of the script and just hardcode it like MongoDB does.
The simple `curl|bash` installer is expected to have hardcoded defaults to make sure the debugging in the community is easier and there is no configuration drift.